StripeConnect/StripeConnectTests/Internal/Webview/MessageHandlers/OpenAuthenticatedWebViewMessageHandlerTests.swift

```diff
@@ -9,20 +9,17 @@
 import XCTest
 
 class OpenAuthenticatedWebViewMessageHandlerTests: ScriptWebTestBase {
-    func testMessageSend() {
-        let expectation = self.expectation(description: "Message received")
```

Comment:
expectation isn't needed because `evaluateOpenAuthenticatedWebView` was changed to async and is now awaited